### ISSUE TITLE
RequestUi's textView doesn't work correctly in Japanese

### ISSUE THREAD

**Issue by shunkit** [zendesk/zendesk_sdk_ios#363]
- SDK version: 2.0.0
- Xcode version: 9.2
- iOS or Simulator: iOS
- iOS Version: 11.2
- Swift version: 4.0.3

Steps to Reproduce:

1. Push RequestUi ViewController
```
let controller = RequestUi.buildRequestUi()
self.navigationController?.pushViewController(controller, animated: true)
```

2. Input some text with Japanese keyboard

```
Input text: ああ
```

3. Input text duplicates while editing

```
TextView's text: あああ
```

I think this problem is the same with below.
https://stackoverflow.com/questions/31430308/uitextview-attributedtext-with-japanese-keyboard-repeats-input

**Comment by divo** [comment]
@shunkit Thanks for reporting this. I have added a bug to our internal backlog. We are working fixing some small UI issues at the moment.

**Comment by fycadw** [comment]
The same problem in Chinese.

**Comment by divo** [comment]
@shunkit @fycadw thank you for your patience. We have just released 2.0.4. It addresses this issue
